### DISCUSSION TITLE
Centralize content of the documentation page

### DISCUSSION THREAD

**Discussion by helderberto** [remix-run/remix#2167] [Proposals]
I noticed that the documentation render in big screens seems like this:
![SCR-20220301-fox](https://user-images.githubusercontent.com/862575/156160016-ee16e475-7b93-4f44-9bf7-36bc2e1368cb.png)
:
![SCR-20220301-fot](https://user-images.githubusercontent.com/862575/156159999-e2ded159-83f5-47fa-9053-7b6196c26a38.png)

A tip I can give is to centralize the content to keep the user's eyes on the center of the screen. What do you think about it?

**Comment by kiliman** [comment]
I typically don't maximize windows on a large screen, but I checked the Tailwind CSS docs site and sure enough, they center the content. Since the Remix docs uses Tailwind, I'm sure it's just a matter of adding a couple of classes to do this.
<img width="3100" alt="image" src="https://user-images.githubusercontent.com/47168/156190754-2b6808c5-419e-465c-b844-3cdd9f6af5e8.png">

**Reply by helderberto** [reply]
Yeah, for sure it is just a couple of classes to adjust.

I'm not maximizing the screen, the screenshots are from the "regular" size of my screen.
